config/initializers/new_framework_defaults_6_1.rb

```diff
@@ -0,0 +1,67 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
We can delete this file as we've set the defaults to be 6.1 in https://github.com/alphagov/collections-publisher/pull/1399/files#diff-c1fd91cb1911a0512578b99f657554526f3e1421decdb9e908712beab57e10f9R25

This file is useful to help you opt into new defaults gradually before switching the default to be 6.1